engine/src/main/scala/cromwell/server/CromwellServer.scala

```diff
@@ -52,16 +52,18 @@ object CromwellServer {
 }
 
 class CromwellServerActor(config: Config) extends CromwellRootActor with CromwellApiService with SwaggerService {
-  implicit def executionContext = actorRefFactory.dispatcher
+  implicit def executionContext: ExecutionContextExecutor = actorRefFactory.dispatcher
 
   override val serverMode = true
   override val abortJobsOnTerminate = false
 
-  override def actorRefFactory = context
-  override def receive = handleTimeouts orElse runRoute(possibleRoutes)
+  override def actorRefFactory: ActorContext = context
+  override def receive: PartialFunction[Any, Unit] = handleTimeouts orElse runRoute(possibleRoutes)
 
-  val possibleRoutes = workflowRoutes.wrapped("api", config.as[Option[Boolean]]("api.routeUnwrapped").getOrElse(false)) ~ swaggerUiResourceRoute
-  val timeoutError = APIResponse.error(new TimeoutException("The server was not able to produce a timely response to your request.")).toJson.prettyPrint
+  val routeUnwrapped: Boolean = config.as[Option[Boolean]]("api.routeUnwrapped").getOrElse(false)
```

Comment:
IntelliJ added this specific ascription and the others nearby. I'm not going to fight IntelliJ until our github has a competing standard to the installed default: https://blog.jetbrains.com/scala/2016/10/05/beyond-code-style/